frontend/public/components/environment.jsx

```diff
@@ -191,6 +191,11 @@ export class EnvironmentPage extends PromiseComponent {
     });
 
     return <div className="co-m-pane__body">
+      { !readOnly &&
+          <div>
+            <p className="co-m-pane__explanation">Define environment variables as key-value pairs to store configuration settings. Enter text or reference a value from a ConfigMap or Secret. Drag and drop the environment variables to change the order they are run in. Lower variables can reference higher ones, for example FULLDOMAIN = $(SUBDOMAIN).example.com</p>
```

Comment:
Maybe "Later variables can reference earlier ones"? The terms lower and higher confused me.

Reply:
And perhaps a `<code>` around the code `FULLDOMAIN = $(SUBDOMAIN).example.com`?